.github/workflows/test.yml

```diff
@@ -55,6 +55,9 @@ jobs:
       run: cargo test ${{ matrix.cargo_profile }} --all-features -- --include-ignored --show-output
       # can not run tests if cpp driver not installed
       if: ${{ matrix.runner == 'ubuntu-18.04' }}
+    - name: Ensure that custom benches run
+      run: cargo run --release --example cassandra_bench -- --config-dir example-configs/cassandra-passthrough --rate 1000
```

Comment:
Does this just check that it still runs?

Reply:
yeah